react/features/e2ee/components/E2EESection.js

```diff
@@ -142,10 +153,23 @@ class E2EESection extends Component<Props, State> {
  * @returns {Props}
  */
 function mapStateToProps(state) {
-    const { enabled } = state['features/e2ee'];
+    const { enabled: toggled, maxMode } = state['features/e2ee'];
+
+    let descriptionResource = '';
+
+    if (maxMode === MAX_MODE_DISABLED) {
+        descriptionResource = 'dialog.e2eeDescription';
+    } else if (maxMode === MAX_MODE_THRESHOLD_EXCEEDED) {
+        descriptionResource = 'dialog.e2eeDisabledDueToMaxModeDescription';
+    } else {
```

Comment:
LEt's reverse the logic please and make the max mode disabled branch the else for easier readability.